README.md

```diff
@@ -11,3 +11,44 @@ A collection of framework specific Auth utilities for working with Supabase.
 
 - [Remix](https://github.com/supabase-community/supabase-auth-helpers/issues/57)
 - [SvelteKit](https://github.com/supabase-community/supabase-auth-helpers/issues/54)
+
+### Apps and Packages
+
+- `nextjs`: a [Next.js](https://nextjs.org) app
+- `@supabase/auth-helpers-nextjs`: the supabase auth helper nextjs library used by `nextjs` application
+- `@supabase/auth-helpers-react`: the supabase auth helper reactjs library used by `nextjs` application
+- `shared`: shared typescript types used by `@supabase/auth-helpers-nextjs` library
+- `config`: `eslint` configurations (includes `eslint-config-next` and `eslint-config-prettier`)
+- `tsconfig`: `tsconfig.json`s used throughout the monorepo
+
+Each package/app is 100% [TypeScript](https://www.typescriptlang.org/).
+
+### Utilities
+
+This turborepo has some additional tools already setup for you:
+
+- [TypeScript](https://www.typescriptlang.org/) for static type checking
+- [ESLint](https://eslint.org/) for code linting
+- [Prettier](https://prettier.io) for code formatting
+
+## Setup
+
+This repository is used in the `npx create-turbo@latest` command, and selected when choosing which package manager you wish to use with your monorepo (PNPM).
```

Comment:
I think this step isn't entirely clear. Do I need to run `npx create-turbo@latest`? Can you make this clearer? how to I install the necessary dependencies?

Reply:
Or are you saying that this repo was set up via `npx create-turbo@latest`? But then do I just have to run pnpm install?

Reply:
I should remove that line because its saying the repository was setup with that command. Yes you would just need to install pnpm and run pnpm install at the root level to get everything installed.

Reply:
Okay, yes, then let's remove it and just say

- Install [pnpm](https://pnpm.io/installation)
- Run `pnpm install`

or something along those lines